### PR TITLE
Limit active reservations.

### PR DESCRIPTION
**Pull request by tuomas777**
Added a limitation on how many active reservations a user can have on a single resource. The limits are imported from kirjasto data. The limits apply only for reservations created via API.

Also updated localisations and added translations for equipment stuff that was missing.

Refs #63

### CODE REVIEW

File: locale/fi/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -17,391 +17,511 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: resources/admin.py:12 resources/admin.py:15
-msgid "RESPA Resource booking system"
-msgstr "RESPA Resurssien varausjärjestelmä"
+#: resources/admin/period_inline.py:38
+msgid "HH:mm"
+msgstr ""
```
Comment (juyrjola) [inline locale/fi/LC_MESSAGES/django.po]: Why is this string translated?

File: locale/fi/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -17,391 +17,511 @@ msgstr ""
...
-msgstr "Katuosoite"
+#: resources/models/availability.py:124
+msgid "periods"
+msgstr "aikavälit"
 
-#: resources/models.py:182
-msgid "Postal code"
-msgstr "Postinumero"
+#: resources/models/availability.py:210
+#, fuzzy
```
Comment (juyrjola) [inline locale/fi/LC_MESSAGES/django.po]: leftover fuzzy here? The strings below seem very much alike, can they be combined?

File: locale/fi/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -17,391 +17,511 @@ msgstr ""
...
+#: resources/models/availability.py:213
+msgid "You must set either 'resource' or 'unit', but not both"
+msgstr "Aseta 'resource' tai 'unit', älä molempia"
 
-#: resources/models.py:184
-msgid "Email"
-msgstr "Sähköpostiosoite"
+#: resources/models/availability.py:255
+msgid "Monday"
+msgstr "Maanantai"
```
Comment (juyrjola) [inline locale/fi/LC_MESSAGES/django.po]: Day of week in Finnish begin with a lower case letter.

File: locale/fi/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -17,391 +17,511 @@ msgstr ""
...
+#: resources/models/availability.py:266
+msgid "Time when opens"
+msgstr "Aukeamisaika"
 
-#: resources/models.py:221
-msgid "unit identifier"
-msgstr "toimipisteen tunniste"
+#: resources/models/availability.py:267
+msgid "Time when closes"
+msgstr "Kiinnimenoaika"
```
Comment (juyrjola) [inline locale/fi/LC_MESSAGES/django.po]: Sulkemisaika?

File: locale/fi/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -17,391 +17,511 @@ msgstr ""
...
+msgid "Category"
+msgstr "Kategoria"
+
+#: resources/models/equipment.py:27 resources/models/equipment.py:28
+msgid "equipment"
+msgstr "varuste"
+
+#: resources/models/equipment.py:41
+msgid "equipment alias"
+msgstr "varustealias"
```
Comment (juyrjola) [inline locale/fi/LC_MESSAGES/django.po]: varusteen toinen nimi?

File: locale/fi/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -17,391 +17,511 @@ msgstr ""
...
+msgid "Time of modification"
+msgstr "Muokkausaika"
+
+#: resources/models/base.py:31
+msgid "Modified by"
+msgstr "Muokkaaja"
+
+#: resources/models/equipment.py:14
+msgid "equipment category"
+msgstr "varustekategoria"
```
Comment (juyrjola) [inline locale/fi/LC_MESSAGES/django.po]: kategoria -> luokka

File: locale/fi/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -17,391 +17,511 @@ msgstr ""
...
-msgstr "Resurssi"
-
-#: resources/models.py:501
-msgid "Begin time"
-msgstr "Alkuaika"
+#: resources/models/resource.py:302
+#, fuzzy
+#| msgid "Main type"
+msgid "Main photo"
+msgstr "Päätyyppi"
```
Comment (juyrjola) [inline locale/fi/LC_MESSAGES/django.po]: Päävalokuva

File: locale/fi/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -17,391 +17,511 @@ msgstr ""
...
 
-#: resources/models.py:502
-msgid "End time"
-msgstr "Loppuaika"
+#: resources/models/resource.py:303
+#, fuzzy
+#| msgid "Ground plan URL"
+msgid "Ground plan"
+msgstr "Pohjapiirroksen osoite"
```
Comment (juyrjola) [inline locale/fi/LC_MESSAGES/django.po]: Pohjapiirros

File: locale/fi/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -17,391 +17,511 @@ msgstr ""
...
+msgstr ""
 
-#: resources/models.py:552
-msgid "reservations"
-msgstr "varaukset"
+#: resources/models/resource.py:310
+#, fuzzy
+#| msgid "Location"
+msgid "Caption"
+msgstr "Sijainti"
```
Comment (juyrjola) [inline locale/fi/LC_MESSAGES/django.po]: Kuvateksti

File: locale/fi/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -17,391 +17,511 @@ msgstr ""
...
+msgstr ""
 
-#: resources/models.py:573
-msgid "Exceptional period"
-msgstr "Poikkeusaikaväli"
+#: resources/models/resource.py:383
+#, fuzzy
+#| msgid "resource type"
+msgid "resource image"
+msgstr "resurssityyppi"
```
Comment (juyrjola) [inline locale/fi/LC_MESSAGES/django.po]: resurssin kuva

File: locale/fi/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -17,391 +17,511 @@ msgstr ""
...
 
-#: resources/models.py:579
-msgid "Start date"
-msgstr "Alkupäivä"
+#: resources/models/resource.py:384
+#, fuzzy
+#| msgid "resource types"
+msgid "resource images"
+msgstr "resurssityypit"
```
Comment (juyrjola) [inline locale/fi/LC_MESSAGES/django.po]: resurssin kuvat

File: locale/fi/LC_MESSAGES/django.po (hunk 1 above)
Comment (tuomas777) [inline locale/fi/LC_MESSAGES/django.po]: hmm, it is shown to (admin) users as a placeholder, why it should not be translated?

**Comment by tuomas777** [comment]
Updated localisations.

**Pull request merged**